cpu/samd21/Kconfig

```diff
@@ -42,15 +29,12 @@ config HAS_CPU_SAMD21
 ## Common CPU symbols
 config CPU_FAM
     default "samd21" if CPU_FAM_SAMD21
-
-config CPU_MODEL
-    default "samd21e18a" if CPU_MODEL_SAMD21E18A
-    default "samd21g18a" if CPU_MODEL_SAMD21G18A
-    default "samd21j18a" if CPU_MODEL_SAMD21J18A
-    default "samr21e18a" if CPU_MODEL_SAMR21E18A
-    default "samr21g18a" if CPU_MODEL_SAMR21G18A
+    default "samr21" if CPU_FAM_SAMD21
```

Comment:
The `CPU_FAM` value in the Makefile should be updated as well.

Reply:
arg, that means we also have to update those `#ifdef CPU_FAM_SAMD21`.
Will `CPU_COMMON_SAMD21` be a define we could test instead?

Reply:
When using Kconfig yes, `CONFIG_CPU_COMMON_SAMD21` would be defined, because of the symbol. Maybe it makes sense to inject that CFLAG from the Makefile, so that when we switch to Kconfig it does not need to be changed?

Reply:
So how do we get `CPU_FAM_SAMD21` and not `CONFIG_CPU_FAM_SAMD21`?